src/dashboard-widgets/dashboard-widgets.tsx

```diff
@@ -3,16 +3,16 @@ import { css } from "@emotion/core";
 import Parcel from "single-spa-react/parcel";
 
 const patientDashboardParcels = [
-  // A patient dashboard widget within this git repo
   () => import("./basic-info/basic-info.parcel").then(m => m.default),
+  () => import("../formentry/forms-parcel").then(m => m.default)
 ];
 
 export default function DashboardWidgets(props: DashboardWidgetsProps) {
   return (
     <div
       css={css`
         margin: 76px auto 0 auto;
-        width: 500px;
+        width: 100%;
```

Comment:
Why did you change this?

Reply:
Was trying something wasn’t  supposed  to push that.